sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -82,6 +89,45 @@ impl EmulatorProcess {
         return EMULATOR_INVALID_ID;
     }
 
+    pub fn get_configurations(&self) -> Result<EmulatorConfig> {
+        let args_list = vec![
+            OsString::from("--path"),
+            self.runtime_directory.clone().into_os_string(),
+            OsString::from("--get_configs"),
+        ];
+
+        let exec: PathBuf = match self.current_args.get("exec") {
+            Some(EmuValue::String(exec_name)) => self.executable_directory.join(exec_name),
+            _ => {
+                bail!(EmuError::RuntimeError(
+                    "Can't get configurations invalid executable".to_string()
+                ))
+            }
+        };
+
+        log::info!("Ti50Emulator getting configuration ");
+        log::info!("Command: {} {:?}", exec.to_string_lossy(), args_list);
```

Comment:
I think `exec.display()` is the method to call here.
Alternatively, you could use the debug specifier:
```
log::info!("Command: {:?} {:?}", exec, args_list);
```